### PR TITLE
add basic notebook

### PR DESCRIPTION
**Pull request by tuanng-cognite**
raster to be added later

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "geospatial-examples"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
```
Comment (pawelzwolenskicognite) [inline pyproject.toml:5]: should be changed :)

File: .github/workflows/pull-request.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        run: |
+          python -m pip install --upgrade pip
+          pip install --no-cache poetry>=1.0.2 twine
+          poetry install
+      - name: Check codestyle
+        run: |
+          poetry run black --check notebooks
+      - name: Test
+        env:
+          COGNITE_CLIENT_SECRET: ${{ secrets.COGNITE_CLIENT_SECRET }}
```
Comment (vincent-cognite) [inline .github/workflows/pull-request.yaml:31]: how do you set up the secrets?

Comment (tuanng-cognite) [reply .github/workflows/pull-request.yaml:31]: In the Security setting, Secret section

Comment (vincent-cognite) [reply .github/workflows/pull-request.yaml:31]: I don't see it. I might not have access to it. Should every member of the geospatial team be added as an "owner" of the repo, maybe?

Comment (tuanng-cognite) [reply .github/workflows/pull-request.yaml:31]: yeap, I'll update some permission/access.

Comment (tuanng-cognite) [reply .github/workflows/pull-request.yaml:31]: added.

Comment (tuanng-cognite) [reply .github/workflows/pull-request.yaml:31]: @vincent-cognite Can you check if access is available?

Comment (vincent-cognite) [reply .github/workflows/pull-request.yaml:31]: yes, I can update or delete secrets under "settings -> secrets" 👍

**Pull request merged**